### ISSUE TITLE
Inference Accuracy about Resnet18 on Imagenet

### ISSUE THREAD

**Issue by Rtr6666** [neurosim/DNN_NeuroSim_V1.4#7]
Why is the accuracy obtained after using this instruction（‘python inference.py --dataset imagenet --model ResNet18 --mode FP --inference 1 --onoffratio 100’） for inference only 10%? Pretrained weights have been introduced in the source code, but why is the accuracy very low?
![QQ20240402-141159](https://github.com/neurosim/DNN_NeuroSim_V1.4/assets/110024745/86cbe0b2-d162-4783-bbc7-4a7dd849c63e)

**Comment by Rosa-Mao** [comment]
Did you solve the problem?

**Comment by littlezhuozhuo** [comment]
Maybe the ADC precision and on/off ratio is too low. It is suggested to set inference = 0 to test the accuracy of the pre-trained network.
